pages/linux/arpaname.md

```diff
@@ -1,6 +1,6 @@
 # arpaname
 
-> Provides corresponding ARPA name for IP addresses.
+> Get the corresponding ARPA name for a IP addresses.
```

Comment:
I think `arpaname` actually supports multiple addresses in the same command.

Reply:
Hmmm, I didn't think about that possibility, we need to check that